src/core/include/openvino/core/descriptor/tensor.hpp

```diff
@@ -27,7 +27,21 @@ using TensorLabelVector = std::vector<TensorLabel>;
 
 namespace pass {
 class ReverseShapeAndTypeInfer;
-}
+class SymbolicPropagation;
+}  // namespace pass
+
+namespace op {
+class TypeRelaxedBase;
+class TemporaryReplaceOutputType;
+
+std::unordered_map<size_t, std::pair<ov::Tensor, ov::Tensor>> OPENVINO_API
+convert_input_types(OutputVector& inputs, const element::TypeVector& types);
+
+void OPENVINO_API
+reset_input_types(const std::unordered_map<size_t, std::pair<ov::Tensor, ov::Tensor>>& original_input_vals,
+                  OutputVector& inputs);
```

Comment:
Do we have to expose these methods?

Reply:
Here just a statement for friend func, following is the original define at here:
https://github.com/openvinotoolkit/openvino/blob/7987fef9ff6e9e0e0ddad57f5d8557cd4e71fad7/src/common/transformations/include/ov_ops/type_relaxed.hpp#L303-L310
Seems like this func only be used in here, maybe we don't need to expose there methods.
https://github.com/openvinotoolkit/openvino/blob/7987fef9ff6e9e0e0ddad57f5d8557cd4e71fad7/src/common/transformations/include/ov_ops/type_relaxed.hpp#L314-L326